bin/kaleidoscope-builder

```diff
@@ -58,23 +58,42 @@ build_filenames () {
 
 enable_ccache () {
     if  [ "$(command -v ccache)" ]; then
-    if ! [ -d "$CCACHE_WRAPPER_PATH" ]; then
-	mkdir -p "$CCACHE_WRAPPER_PATH"
-	ln -s "$(command -v ccache)" "${CCACHE_WRAPPER_PATH}/${COMPILER_PREFIX}gcc"
-	ln -s "$(command -v ccache)" "${CCACHE_WRAPPER_PATH}/${COMPILER_PREFIX}g++"
-	ln -s "${AVR_NM}" "${CCACHE_WRAPPER_PATH}/${COMPILER_PREFIX}nm"
-	ln -s "${AVR_OBJCOPY}" "${CCACHE_WRAPPER_PATH}/${COMPILER_PREFIX}objcopy"
-	ln -s "${AVR_AR}" "${CCACHE_WRAPPER_PATH}/${COMPILER_PREFIX}ar"
-	ln -s "${AVR_SIZE}" "${CCACHE_WRAPPER_PATH}/${COMPILER_PREFIX}size"
-    fi
-    export CCACHE_PATH=${COMPILER_PATH}/
-    CCACHE_ENABLE="-prefs compiler.path=${CCACHE_WRAPPER_PATH}/"
+      if ! [ -d "$CCACHE_WRAPPER_PATH" ]; then
```

Comment:
The following changes make it possible to use ccache for virtual and non-virtual builds in parallel.